Cargo.lock

```diff
@@ -2765,19 +2765,24 @@ dependencies = [
  "async-trait",
  "atty",
  "clap 4.5.20",
+ "criterion",
```

Comment:
Should this be a dev dependency?

Reply:
yeap this is declared as dev dependency in `forc-crypto` `Cargo.toml`